curriculum/challenges/english/25-front-end-development/lab-luhn-algorithm/68507cadbf36aa089a84ce2d.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 68507cadbf36aa089a84ce2d
+title: Implement the Luhn Algorithm
+challengeType: 27
+dashedName: implement-the-luhn-algorithm
+---
+
+# --description--
+
+The Luhn algorithm, also known as the "modulus 10" or "mod 10" algorithm, is a simple checksum formula used to validate a variety of identification numbers, like credit card numbers. These are the steps to validate a number using the Luhn algorithm:
+
+- Starting from the right, and excluding the rightmost digit (the check digit), double the value of every other digit.
+- If the result of doubling a digit is greater than `9`, sum the digits to get a single digit. Alternatively, you can subtract `9` from the result.
+- Take the sum of all the digits including the check digit.
+- If the sum of all the digits is a multiple of `10`, then the number is valid; else it is not valid.
+
+Let's say we have the number `453914880`. The steps to validate it using the Luhn algorithm would be:
+
+```md
+Account number       4   5   3   9   1   4   8   8   0   
+Double every other       ✔       ✔       ✔       ✔   x 
+                     4   10  3   18  1   8   8   16  0  
+Sum 2-char digits    4    1  3    9  1   8   8    7  0
+Total Sum:           4 + 1 + 3 + 9 + 1 + 8 + 8 + 7 + 0 = 41
+```
```

Comment:
I don't like how this looks rendered, `✔` is not monospaced, I think we need to find a different character
![image](https://github.com/user-attachments/assets/819bc3ac-5cc4-4c74-95dd-6ca6b0dfab10)

Reply:
Is there another font that we should potentially add that covers the character?